README.md

```diff
@@ -1,8 +1,8 @@
 # Office Scripts documentation
 
-Welcome to the Office Scripts documentation repository. In this repo, you can find the documentation source files for Office Scripts tutorials and how-to guides. For the best experience, we recommend you view this content on [docs.microsoft.com](https://docs.microsoft.com/office/dev/scripts).
+Welcome to the Office Scripts documentation repository. In this repo, you can find the documentation source files for Office Scripts tutorials and how-to guides. For the best experience, we recommend you review the [Office Scripts documentation](https://docs.microsoft.com/office/dev/scripts).
```

Comment:
This changes the meaning of the line. This GitHub readme is intended to tell people not to view the docs in GitHub itself, but rather go to the rendered DMC site.

Reply:
That makes sense. "docs.microsoft.com" is supposed to be avoided as sentence text or as a link description - can you think of a different phrase we could use here that wouldn't change the meaning?

#docutune-wait

Reply:
Thanks for understanding and explaining the changes. How about something like this?
```suggestion
Welcome to the Office Scripts documentation repository. In this repo, you can find the documentation source files for Office Scripts tutorials and how-to guides. For the best experience, we recommend you view this content on [Microsoft Docs](https://docs.microsoft.com/office/dev/scripts).
```
(I'll be sure to change the text once the Learn rebrand happens.)